noir-projects/noir-contracts/contracts/account/schnorr_account_contract/src/main.nr

```diff
@@ -56,7 +56,7 @@ pub contract SchnorrAccount {
 
         // Safety: Comment from above applies here as well.
         unsafe { set_sender_for_tags(self.address) };
-        self.storage.signing_public_key.at(self.address).initialize(pub_key_note).deliver(
+        self.storage.signing_public_key.initialize(pub_key_note).deliver(
```

Comment:
Hm, this actually works almost accidentally. Because we set the contract to be the note owner, here `deliver` will deliver the message to that address, which by being the acct contract happens to work. Perhpas it'd be better to more explicitly do `deliver_to(self.addr)`?